src/blocks/time.rs

```diff
@@ -42,7 +39,6 @@ pub struct Config {
     #[default(1.into())]
```

Comment:
We can now use normal `FormatConfig` instead of `DummyConfig`. I think we can even remove `DummyConfig`.

Reply:
github doesn't allow comments on unedited lines for some reason :angry: